dlup/backends/tifffile_backend.py

```diff
@@ -47,9 +47,9 @@ def __parse_tifffile(self) -> None:
         for idx, page in enumerate(self._image.pages):
             # Remove channel dimension and swap rows and columns
             if len(page.shape) == 3:
-                self._shapes.append((page.shape[1], page.shape[2]))
+                self._shapes.append((page.shape[1], page.shape[0]))
```

Comment:
Did you test this? In the documentation of tifffile it says that a Tiffpage has Shape ([depth,] length, width)
https://github.com/cgohlke/tifffile/blob/master/tifffile/tifffile.py#L7729

Reply:
has been checked with stored tif image and the channel dimension came indeed last, so the change is correct